common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' aria-hidden='true'><i class='icon fa fa-times'></i></button>" +
+        "  <button class='close-modal dismiss' aria-hidden='true'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
One more thing for knowledge. Anything that receives the `aria-hidden="true"` attribute won't be seen or read by screenreaders. In this case, our button is inaccessible. Let's remove the `aria-hidden` from the button, and make sure the icon has it. You can either use a `title` on the button, or our traditional `sr` text method. Does this make sense?